### DISCUSSION TITLE
Version update doubles the database in the cache

### DISCUSSION THREAD

**Discussion by Scorter** [realm/realm-dotnet#3646] [Q&A] [answered]
Hello everyone, I recently updated the realms version in one of my Unity projects from version 11.7.0 to version 12.2.0. The problem I have encountered is that there has been a duplication of files in the cache, practically doubling the amount of storage used, and this can complicate situations where some user data is very large. I would like to know if this is normal after a version update or am I forgetting something? Follow the attachments to see the before and after
![realmsbeforeupdated](https://github.com/realm/realm-dotnet/assets/88690064/95f1373b-d7ac-4ad5-8228-e583c12dfcb5)
![realmsupdated](https://github.com/realm/realm-dotnet/assets/88690064/d4481215-2e44-4297-a467-8642d7c2ba51)

**Comment by nirinchev** [comment, marked as answer]
The .v23 backup realm is a backup we create because we need to bump the version of the database. If you need to rollback an app update to a version that uses an older Realm version, it wouldn't be able to open the upgraded database, which is why the backup is there. If you are confident this is not a scenario you want to cover, you can delete the backup.

As a note, the backup file is retained for 3 months, so Realm will automatically delete it if it's not used after 3 months.
